src/storekit.cs

```diff
@@ -674,4 +719,13 @@ interface SKProductDiscount {
 		[Export ("paymentMode")]
 		SKProductDiscountPaymentMode PaymentMode { get; }
 	}
+
+	[iOS (11,3), TV (11,3), NoMac]
+	[BaseType (typeof (NSObject))]
```

Comment:
add `[DisableDefaltCtor]` since there's no apparent reason why `init` should be called

Reply:
ahh nice catch, damn it missed it 😛

Reply:
@spouliot fixed it, thanks!